pandas/tests/io/test_sql.py

```diff
@@ -2525,30 +2525,16 @@ def test_schema_support(self):
             tm.assert_frame_equal(res1, res2)
 
 
-@pytest.mark.db
-class TestMySQLAlchemy(_TestMySQLAlchemy, _TestSQLAlchemy):
```

Comment:
So now that connections are always made internally, these tests are redundant?

Reply:
They no longer test what would actually happen internally, since they pass an Engine to SQLDatabase.  Instead, TestMySQLAlchemyConn tests passing a Connection.